gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https://services.gradle.org/distributions/gradle-8.0.2-bin.zip
-distributionSha256Sum=ff7bf6a86f09b9b2c40bb8f48b25fc19cf2b2664fd1d220cd7ab833ec758d0d7
+distributionUrl=https://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
This is because gradle 8.0.2 had a problem with dynamic tests: https://github.com/gradle/gradle/pull/23285

Reply:
The reason this won't work is that the hash that we're checking against also needs to be updated.